frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -242,3 +242,21 @@ plans:
       dn-deploy:
         strategy: {{DATA_STRATEGY}}
         pod: data
+
+  replace-nn-0:
```

Comment:
Since we have to write custom Java for this anyway, couldn't it construct two separate `Plan` instances from the same raw YAML, so there could just be a single `replace-nn` plan here?